shared/constants/teams.js

```diff
@@ -175,11 +175,8 @@ const userIsActiveInTeamHelper = (
   return member && member.active
 }
 
-const getConvIdsFromTeamName = (state: TypedState, teamname: string): I.Set<ChatTypes.ConversationIDKey> =>
-  I.Set(getTeamChannelInfos(state, teamname).keys())
-
-const getBadgeSubscribe = (state: TypedState, teamname: string): boolean =>
```

Comment:
badge subscribe seems like a UI-level concept, so I think it's better to keep these functions at the level of the store